entrypoint.sh

```diff
@@ -99,11 +100,15 @@ validate() {
 }
 
 write_back() {
+  # The configuration_output_branch is optional. If not set, the
+  # write back branch will be the same as the target branch.
+  write_back_branch=${configuration_output_branch:-$target_branch}
```

Comment:
This syntax will use `target_branch` if the user does not set `configuration_output_branch` explicitly. This includes `configuration_output_branch` being `""` or not present at all.